### DISCUSSION TITLE
Bulk Download

### DISCUSSION THREAD

**Discussion by Wjxfi** [ente-io/ente#969] [Enhancements]
There should be a download button in place of the orange circle
![300946.jpg](https://user-images.githubusercontent.com/113500363/212191041-2833f959-28da-4734-af5f-96023a75d1a9.jpg)

**Comment by mnvr** [comment]
Hey! So the reason that option is not there because that option already exists in the system share sheet (that is invoked by pressing the second button) e.g. on iOS there is an option to "Save files", and on Android, the default gallery app usually shows that as option.

However, it is possible that your gallery app might not be providing that option. If that's the case, please feel free to open a feature request on https://roadmap.ente.io - we (roughly) take the upvotes there to prioritize which tasks to work on first!

**Comment by Wjxfi** [comment]
sharing is a bad solution. i need to download an additional app to download and everything will be downloaded in one folder. i need it to download to the right folders. where does the ente app get the photos or videos i want
(I don't have a file saving feature)

**Comment by mnvr** [comment]
It is not really sharing, e.g. on iOS it is just asking the system app to save it for us.

But yes, if you're not comfortable doing that with your system app (or don't have a file saving feature), then you can download the file on the photo screen. 

There is an option to download the photo on the photo details. I know you're looking for an option to save multiple at the same time, but at least, this way you are not meanwhile blocked for your work.

**Comment by Wjxfi** [comment]
blocked?

**Comment by mnvr** [comment]
I meant that you're not prevented from saving/downloading the file, just that you'll need to do it one by one.
